user-interface/src/data-verification/ConsolidationOrderModal.tsx

```diff
@@ -33,22 +33,49 @@ export interface ConsolidationOrderModalProps {
   onConfirm: (results: ConfirmActionResults) => void;
 }
 
+type ConfirmationSteps = 'pick-lead-case' | 'confirm';
+
 type ShowOptionParams = {
   status: OrderStatus;
-  caseIds: string[];
-  attorneys: AttorneyInfo[];
+  cases: ConsolidationOrderCase[];
 };
 
 type ShowOptions = {
   status: OrderStatus;
   heading: string;
-  attorneys: AttorneyInfo[];
 };
 
 export type ConfirmationModalImperative = ModalRefType & {
   show: (options: ShowOptionParams) => void;
 };
 
+export async function getCaseAssignments(caseId: string): Promise<Array<CaseAssignment>> {
+  try {
+    const api = useApi();
+    const response = await api.get(`/case-assignments/${caseId}`);
+    return response.body as CaseAssignment[];
+  } catch {
+    // TODO: If this API call fails because the case ID cannot be found then this is an invalid case.
```

Comment:
Need to consider if we can confirm the approval if this API call fails. We should return a 404 for a case that isn't found. In that condition the lead case doesn't exist and as such is invalid to confirm the approval.

Reply:
Code for this is in branch `CAMS-329-assignment-BRIAN`

Reply:
Merged the two branches.